### ISSUE TITLE
Hello

### ISSUE THREAD

**Issue by lgc-1997** [simongu20070911/thinkpad-t14-gen1-i7-10510U-hackintosh-efi#2]
Complete EFI files. Thank you very much. Can I ask you another question? Is your computer easy to configure? Is the screen resolution 4K? My computer can't be installed with your EFI. Thank you

**Comment by simongu20070911** [comment]
yes.in opencore, the resolution is max, so you will get the highest resolution your thinkpad can get.  Of course, if you find the text too small but don't want to change the resolution, you can use the hiDPi tool, in GitHub.

**Comment by simongu20070911** [comment]
Did you get your efi folder into a fat32 partition? If not, it wont boot.

**Comment by simongu20070911** [comment]
Also, i recommend you use disk genius to create a startup path

**Comment by lgc-1997** [comment]
Yes, I put it in the FAT32 folder. There is your startup theme for startup, and the code gets stuck during installation. So, I want to ask if your screen is 4K?


醉挽城
***@***.***



&nbsp;




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "simongu20070911/thinkpad-t14-gen1-i7-10510U-hackintosh-efi"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2022年6月27日(星期一) 上午9:05
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [simongu20070911/thinkpad-t14-gen1-i7-10510U-hackintosh-efi] Hello (Issue #2)





 
Also, i recommend you use disk genius to create a startup path

**Comment by lgc-1997** [comment]
thank you




醉挽城
***@***.***



&nbsp;




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "simongu20070911/thinkpad-t14-gen1-i7-10510U-hackintosh-efi"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2022年6月27日(星期一) 上午9:03
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [simongu20070911/thinkpad-t14-gen1-i7-10510U-hackintosh-efi] Hello (Issue #2)





 
yes.in opencore, the resolution is max, so you will get the highest resolution your thinkpad can get.  Of course, if you find the text too small but don't want to change the resolution, you can use the hiDPi tool, in GitHub.

**Comment by simongu20070911** [comment]
no ,no, my thinkpad is 1920*1080. you can change the resolution in config.plist, I think you can check the dirtania guide to see where to change the screen resolution

**Comment by lgc-1997** [comment]
ok，ok，Then I know why I can't install it. The screen resolution is different, which leads to the installation failure. Thank you


醉挽城
***@***.***




&nbsp;




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "simongu20070911/thinkpad-t14-gen1-i7-10510U-hackintosh-efi"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2022年6月27日(星期一) 中午11:11
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [simongu20070911/thinkpad-t14-gen1-i7-10510U-hackintosh-efi] Hello (Issue #2)





 
no ,no, my thinkpad is 1920*1080. you can change the resolution in config.plist, I think you can check the dirtania guide to see where to change the screen resolution
